Index: SpringEcommerce/src/main/java/com/ProvaGrupo/SpringEcommerce/service/CartSevice.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.ProvaGrupo.SpringEcommerce.service;public class CartSevice {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/SpringEcommerce/src/main/java/com/ProvaGrupo/SpringEcommerce/service/CartSevice.java b/SpringEcommerce/src/main/java/com/ProvaGrupo/SpringEcommerce/service/CartSevice.java
--- a/SpringEcommerce/src/main/java/com/ProvaGrupo/SpringEcommerce/service/CartSevice.java	(revision 040d889c66eaffffc6133be1242a1ea5170b5bfe)
+++ b/SpringEcommerce/src/main/java/com/ProvaGrupo/SpringEcommerce/service/CartSevice.java	(date 1716390537275)
@@ -1,2 +1,8 @@
-package com.ProvaGrupo.SpringEcommerce.service;public class CartSevice {
+package com.ProvaGrupo.SpringEcommerce.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class CartSevice {
+
 }
